index.test.js

```diff
@@ -50,7 +68,7 @@ it('should initialise IPFS API by accessing it directly when running inside of i
   }
 
   const store = composeBundlesRaw(
-    ipfsBundle()
+    ipfsBundle({ tryApi: false, tryJsIpfs: false })
   )()
 
```

Comment:
Please add tests for 

- `Should connect via js-ipfs-api`
- `Should connect via js-ipfs`
- `Should connect via js-ipfs-api when window.ipfs present but disabled by options`
- `Should connect via js-ipfs-api when window.ipfs present but fails`
- `Should connect via js-ipfs instead of window.ipfs or js-ipfs-api`